src/test/integration/util/common.ts

```diff
@@ -81,6 +81,13 @@ export class CommonUICreator {
 			setTimeout(resolve, 2000);
 		});
 	}
+
+	public async configureSetting(section: string, settingName: string, value: string): Promise<void> {
+		const editor = await this.workbench.openSettings();
+
+		const setting = await editor.findSetting(settingName, `Titanium › ${section}`) as TextSetting;
```

Comment:
Note to self, we can replace this manual handling of nesting here when the next version of vscode-extension-tester is published